.gitlab/e2e_install_packages/installer.yml

```diff
@@ -0,0 +1,15 @@
+qa_installer_script_main:
+  image: registry.ddbuild.io/ci/datadog-agent-buildimages/gitlab_agent_deploy$DATADOG_AGENT_BUILDIMAGES_SUFFIX:$DATADOG_AGENT_BUILDIMAGES
```

Comment:
pinging @chouetz since he's in the process of using per-images variables, this will need to be updated in his PR

Reply:
LGTM, I will have to re-run my script, thanks for checking this